docker-compose.yml

```diff
@@ -11,35 +8,39 @@ services:
       - postgres
       - redis
     volumes:
-      - "./build/data:/data"
+      - "./data/data:/data"
     tmpfs:
       - "/tmp"
-    env_file:
-      - envfile.env
+    environment:
+      BALKHASH_BACKEND: POSTGRESQL
+      BALKHASH_DATABASE_URI: postgresql://opensanctions:opensanctions@postgres/opensanctions
 
   postgres:
     image: postgres:11.4
     volumes:
-      - local_postgres_data:/var/lib/postgresql/data
-    env_file:
-      - envfile.env
+      - "./data/postgres-data:/var/lib/postgresql/data"
+    environment:
+      POSTGRES_USER: opensanctions
+      POSTGRES_PASSWORD: opensanctions
 
```

Comment:
Need to add `POSTGRES_DB: opensanctions`

Reply:
It just defaults to the username if no DB name is given, which seems OK here.